app/proguard-rules.pro

```diff
@@ -24,3 +24,6 @@
 -dontobfuscate
 -keep class com.jcraft.jsch.**
```

Comment:
Just a note, this can go

Reply:
Not yet, unfortunately, unless I missed something. This commit does not fully replace `Jsch`, we will have two SSH backends in parallel for some time.

Reply:
And that's what I get for reading two files on the go